internal/pkg/validator/validator.go

```diff
@@ -289,9 +302,13 @@ func prefixErrorWithNamespace(e validator.FieldError, errMsg string, customNames
 		namespaceParts = strings.Split(errNamespace, ".")
 	}
 
-	// Is field present at the beginning of the error message?
-	errMsgFirstWord := strings.SplitN(errMsg, " ", 2)[0]
+	// Is field present at the beginning of the error message? Then remove it.
+	errMsgParts := strings.SplitN(errMsg, " ", 2)
+	errMsgFirstWord := errMsgParts[0]
 	errMsgContainsField := len(namespaceParts) > 0 && namespaceParts[len(namespaceParts)-1] == errMsgFirstWord
+	if errMsgContainsField {
+		errMsg = errMsgParts[1]
+	}
```

Comment:
Tu a nizsie je vyriesene, ze namiesto `name is a required field` je `"name" is a required field`, takto sa to lepsie cita, pri niektorych nazvoch fields nebolo jasne, ze prve slovo je nazov fieldu.